packages/components/src/components/rating/rating.css

```diff
@@ -0,0 +1,61 @@
+/**
+* @prop --rating__container-display: Display of the rating container
+* @prop --rating__item--star-emoji-display: Display of the rating star rating emoji
+* @prop --rating__item--star-emoji-margin-right: Margin right of the rating star rating emoji
+* @prop --rating__item--star-emoji-cursor: Cursor of the rating star rating emoji
+* @prop --rating__item--star-emoji-font-size: Font size of the rating star rating emoji
+* @prop --rating__item--star-emoji-transition: Transition of the rating star staring emoji
+* @prop --rating__item--star-after-content: Content of the rating star after
+* @prop --rating__item--emoji-after-content: Content of the rating emoji after
+* @prop --rating__item--star-selected-after-color: Color of the rating star selected
+* @prop --rating__item--star-selected-after-content: Content of the rating star selected after
+* @prop --rating__item--emoji-selected-1-after-content: Content of the rating star selected 1 after
+* @prop --rating__item--emoji-selected-2-after-content: Content of the rating star selected 2 after
+* @prop --rating__item--emoji-selected-3-after-content: Content of the rating star selected 3 after
+* @prop --rating__item--emoji-selected-4-after-content: Content of the rating star selected 4 after
+* @prop --rating__item--emoji-selected-5-after-content: Content of the rating star selected 5 after
+
+*/
+
+.rating__container {
+  display: var(--rating__container-display, flex);
+}
+
+.rating__item--star,
+.rating__item--emoji {
+  display: var(--rating__item--star-emoji-display, inline);
+  margin-right: var(--rating__item--star-emoji-margin-right, 6px);
+  cursor: var(--rating__item--star-emoji-cursor, pointer);
+  font-size: var(--rating__item--star-emoji-font-size, 24px);
+  transition: var(--rating__item--star-emoji-transition, color 0.2s);
+}
+
+.rating__item--star:after {
+  content: var(--rating__item--star-after-content, '☆');
```

Comment:
The unicode star is buggy with mac os 10.13 Chrome and Safari. Better to use external icons and not rely on hardcoded unicode characters